src/components/seller/components/sellerItems/sellerItems.tsx

```diff
@@ -0,0 +1,154 @@
+import { ChangeEvent, useEffect, useState } from 'react';
+import styles from './sellerItems.module.scss';
+import CardItem from '../cardItem/cardItem';
+import NotFoundSearch from '../../../../designComponents/notFoundSearch/notFoundSearch';
+import useFetch from '../../../../hooks/useFetch';
+import LoadingSpinner from '../../../../designComponents/loadingSpinner/loadingSpinner';
+import { useDispatch, useSelector } from 'react-redux';
+import { setPage, setSearch } from './sellerItemsFilters.slice';
+import { RootState } from '../../../../redux/store';
+import axios from 'axios';
+
+//defines data
+export interface cardDetails {
+  products: {
+    productImages: { url: string }[];
+    details?: string;
+    id: string;
+    category: string;
+    ratings?: number;
+    price: number;
+    bonus: number;
+    collectionId: string;
+    createdAt?: string;
+    expDate: string;
+    expiredflag: boolean;
+    name: string;
+    quantity: number;
+  }[];
+  message: string;
+  page: number;
+  totalPages: number;
+}
+
+export default function SellerItems() {
+  const dispatch = useDispatch();
+  //redux get initial values from redux
+  const search = useSelector((state: RootState) => state.filterCollectionProducts.search);
+  const page = useSelector((state: RootState) => state.filterCollectionProducts.page);
+  //get instant data from db
+  const url = `https://team-sostene-e-commerce-bn-production-f4c1.up.railway.app/products/all?page=${page}&limit=10`;
```

Comment:
Use env variables here

Reply:
this receives parameters plz, so it dynamic

Reply:
what do mean you can use the environment variable

Reply:
https://team-sostene-e-commerce-bn-production-f4c1.up.railway.app this url might change